dbpedia/dbpedia_Classical_musicians_by_century.js

```diff
@@ -31,180 +30,183 @@ function replaceURLAndUnderscore(str) {
 }
 
 function fetch1(link1) {
```

Comment:
So what does fetch1 do? Naming this function a bit better would give other people an insight into what is supposed to happen here